ecr-login/config/cache_dir.go

```diff
@@ -19,5 +19,7 @@ func GetCacheDir() string {
 	if cacheDir := os.Getenv("AWS_ECR_CACHE_DIR"); cacheDir != "" {
 		return cacheDir
 	}
-	return "~/.ecr"
+	dir, _ := os.UserCacheDir()
```

Comment:
Let's not ignore the error here.

```suggestion
	dir, err := os.UserCacheDir()
	if err != nil {
		return "~/.ecr"
	}
```